service-search-nls/src/main/java/fi/nls/oskari/search/util/ScoredSearchHit.java

```diff
@@ -3,14 +3,20 @@
 class ScoredSearchHit implements Comparable<ScoredSearchHit> {
     public final String text;
     public final int score;
+    public final Boolean isExact;
```

Comment:
isExact could be primitive boolean. You can use Boolean.compare(boolean a, boolean b); for comparisons if that's the reason you chose the Object version.